### PR TITLE
Search devices through SSDP, add basic support to Wemo Insight

### PR DESCRIPTION
**Pull request by luca776**
Instead of maintaining the list of MAC addresses, the search is performed through the SSDP protocol.
Insight class add support to "standby?" state and can determine the power consumption of the device.

### CODE REVIEW

**Comment by gisikw** [comment]
Thanks for the PR! I unfortunately don't have devices to quickly test with, and it's been a while since I've used this. I have a vague recollection of running into problems with SSDP when running on JRuby. If it's not too much trouble, mind verifying that this works in that environment?

File: wemote.gemspec [hunk 1]
```diff
@@ -18,4 +18,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
+
+  spec.add_dependency 'ssdp'
```
Comment (gisikw) [inline wemote.gemspec]: Can we add a version here please? Thanks!

**Comment by luca776** [comment]
Ok, you were right, there was an issue with JRuby, I fixed it on SSDP and the author was very fast to merge my Pull Request. Now I added the dependency to the new gem version (>=1.1.6) and I removed some duplicated code on Insight sublclass.

**Comment by luca776** [comment]
@gisikw do you think it's okay?

**Comment by gisikw** [comment]
So sorry for the delay on this. Thanks for getting it working with JRuby - excited not to be shelling out :)

**Pull request merged**